plugins/tracker/src/index.ts

```diff
@@ -258,6 +258,8 @@ export interface IssueTemplate extends Doc, IssueTemplateData {
   // Discussion stuff
   comments: number
   attachments?: number
+
+  relations?: RelatedDocument[]
```

Comment:
Why we need relations for template? I supposed we will have IssueTemplate instances per Vacancy template, it we will add normal relations for created issues.

Reply:
If we want to create 'IssueTemplates' field in Vacancy template (KanbanTemplate interface), we should add 'tracker' dependency, but this solution doesnt need it